packages/access-client/src/agent-use-cases.js

```diff
@@ -115,38 +115,37 @@ export async function pollAccessClaimUntil(
   opts
 ) {
   const interval = opts?.interval || 250
-  /** @type {ReturnType<typeof setTimeout>} */
-  let pollingTimeoutId
   const claimed = await new Promise((resolve, reject) => {
     opts?.signal?.addEventListener('abort', (e) => {
-      pollingTimeoutId && clearTimeout(pollingTimeoutId)
-      reject(
-        new Error('pollAccessClaimUntilSessionProof aborted', { cause: e })
-      )
+      reject(new Error('pollAccessClaimUntil aborted', { cause: e }))
```

Comment:
I'm not sure we should put function names in error messages. These kinds of things often get shown to users. Can we put a human readable message and let the stack trace take care of identifying the origin?